website/docs/components/badge/index.md

```diff
@@ -13,8 +13,8 @@ links:
 </section>
 
 <section data-tab="Code">
-  @include "partials/code/component-api.md"
   @include "partials/code/how-to-use.md"
+  @include "partials/code/component-api.md"
```

Comment:
I can't comment on the correct line (26), but we should also add the accessibility support file as well (see how alert does it)

Reply:
There weren't any listed in the original docs though: https://design-system-components-hashicorp.vercel.app/components/badge#accessibility

@MelSumner Are there any standard ones I should list in this case?

Reply:
To be more clear: https://github.com/hashicorp/design-system/blob/main/website/docs/components/alert/partials/accessibility/support.md is what I was referring to

Reply:
Oh thanks. Sorry for misunderstanding.

Reply:
Added and pushed up.